internal/generator/backend/main.go

```diff
@@ -102,6 +109,22 @@ func main() {
 				panic(err)
 			}
 
+			// groth16 & plonk
+			if d.Curve == "tinyfield" {
```

Comment:
Maybe instead of manually checking here add field `noBackend` field to `templateData` and check it?